src/lib.rs

```diff
@@ -366,4 +428,21 @@ mod test {
     fn fbm_3d_values_unchanged() {
         assert_debug_snapshot!(sample_3d_fn(|p| { fbm_simplex_3d(p, 5, 2.0, 0.5) }));
     }
+
+    // Enable for seeded version
+    // #[test]
+    // fn worley_2d_values_unchanged() {
```

Comment:
Would be great to have a test, why is it disabled?

Reply:
It produces a different output every time, because it doesn't support seeding yet 😅